templates/common/.github/workflows/java/terraform/azure-dev.yml

```diff
@@ -28,6 +28,24 @@ jobs:
       - run: |
           chmod +x ${GITHUB_WORKSPACE}/src/api/mvnw
 
+      - name: Install Nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
Why install node for java?

Reply:
We always need `node` to build the front-end application, regardless of the api-language

Reply:
ah